_pytest/runner.py

```diff
@@ -315,7 +316,8 @@ class TestReport(BaseReport):
     """
 
     def __init__(self, nodeid, location, keywords, outcome,
-                 longrepr, when, sections=(), duration=0, **extra):
+                 longrepr, when, user_properties,
```

Comment:
I think `user_properties` should go after `duration` to avoid breaking clients needlessly.